README.md

```diff
@@ -120,20 +142,70 @@ The global state reflects the logical _AND_ of each inidvidual sub-state.
 
 ##### Example Status field of a NICClusterPolicy instance
 ```
-Status:                                    
+Status:
   Applied States:
     Name:   state-OFED
     State:  ready
     Name:   state-RDMA-device-plugin
     State:  ready
     Name:   state-NV-Peer
     State:  ignore
+    Name:   state-macvlan-cni
+    State:  ignore
+    Name:   state-Multus
+    State:  ready
+    Name:   state-whereabouts
+    State:  ready
   State:    ready
 ```
 
 >__NOTE__: An `ignore` State indicates that the sub-state was not defined in the custom resource
 > thus it is ignored.
 
+### MacvlanNetwork CRD
+This CRD defines a MacVlan secondary network. It is translated by the Operator to a `NetworkAttachmentDefinition` instance as defined in [k8snetworkplumbingwg/multi-net-spec](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+
+#### MacvlanNetwork spec:
+MacvlanNetwork CRD Spec includes the following fields:
+- `networkNamespace`: Namespace for NetworkAttachmentDefinition related to this MacvlanNetwork CRD.
+- `master`: Name of the host interface to enslave. Defaults to default route interface.
+- `mode`: Mode of interface one of "bridge", "private", "vepa", "passthru", default "bridge".
+- `mtu`: MTU of interface to the specified value. 0 for master's MTU.
+- `ipam`: IPAM configuration to be used for this network.
+
+##### Example for MacvlanNetwork resource:
+In the example below we deploy MacvlanNetwork CRD instance with mode as bridge, mtu 1500, default route interface as master, with resouce "rdma/hca_shared_devices_a", that will be used to deploy NetworkAttachmentDefinition for macvlan to default namespace.
+
+```
+apiVersion: mellanox.com/v1alpha1
+kind: MacvlanNetwork
+metadata:
+  name: example-macvlannetwork
+spec:
+  networkNamespace: "mlnx-network-operator-resources"
```

Comment:
networkNamespace should be `default` or something else, this namespace is reserved for resources the operator deploys, it is unlikely the user would want to define a network in this namespace as its internally used by the operator.

Reply:
Changed to default